### ISSUE TITLE
Make extends tslint.json easier by adding "rulesDirectory": "./".

### ISSUE THREAD

**Issue by HamletDRC** [Microsoft/tslint-microsoft-contrib#268]
branched from #265

**Comment by HamletDRC** [comment]
This is on the releases branch. Can you please test it if you have some time  @dsifford

**Comment by dsifford** [comment]
Do you have a `@next` or `@dev` branch that I can pull from npm or do I have to clone this repo? 

If the latter is true, can you possibly consider adding a dev branch to make it easier for things like this?

**Comment by dsifford** [comment]
Also -- Changes work for me. 👍

**Comment by HamletDRC** [comment]
The branch is called releases. You can reference it in your package.json

**Comment by HamletDRC** [comment]
Great news!

**Comment by dsifford** [comment]
```sh
$ npm install -D tslint-microsoft-contrib@releases
npm ERR! Linux 4.7.4-1-ARCH
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-D" "tslint-microsoft-contrib@releases"
npm ERR! node v6.2.0
npm ERR! npm  v3.10.3

npm ERR! No compatible version found: tslint-microsoft-contrib@releases
npm ERR! Valid install targets:
npm ERR! 2.0.12, 2.0.11, 2.0.10, 2.0.9, 2.0.8, 2.0.7, 2.0.6, 2.0.5, 2.0.3, 2.0.2, 2.0.1, 2.0.0, 1.0.0, 0.0.4, 0.0.3, 0.0.1
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
```
Is there some other way to do this that I'm not aware of? I've installed from "tag" plenty of times, but never from a branch. 

Thanks!

**Comment by HamletDRC** [comment]
Releases is not in non, just git. So the syntax is like https://.../tslint-microsoft-contrib@releases or something. I am on mobile now, let me know if you can't get it working.

**Comment by dsifford** [comment]
Good to know. Sorry for derailing this, but figured that little gem would be nice to have in my back pocket 😄 

Thanks again!

**Comment by dsifford** [comment]
Aha! This was it...

```
npm i -D git://github.com/Microsoft/tslint-microsoft-contrib.git#releases
```
